webapp/routes.go

```diff
@@ -44,6 +44,9 @@ func RegisterRoutes() {
 	// Dashboard for the compat-2021 effort.
 	shared.AddRoute("/compat2021", "compat-2021", compat2021Handler)
 
+	// Dashboard for the interop-2022 effort.
+	shared.AddRoute("/interop2022", "interop-2022", interop2022Handler)
```

Comment:
Can you add a dash here and anywhere else it's missing? There's no particular reason to have it or not, but the repo for the effort has it.

Reply:
Got on the computer to suggest instead:

```suggestion
	shared.AddRoute("/interop-2022", "interop-2022", interop2022Handler)
```